test/integration/aws_lambda_filter_integration_test.cc

```diff
@@ -18,7 +18,8 @@ name: io.solo.aws_lambda
 const std::string USE_CHAIN_LAMBDA_FILTER =
     R"EOF(
 name: io.solo.aws_lambda
-config:
+typed_config:
+  "@type": type.googleapis.com/envoy.config.filter.http.aws_lambda.v2.AWSLambdaConfig
```

Comment:
can we do v3 here, as it's filter config rather than top-level?

Reply:
this is our own config, so v3 doesn't apply here